frontend/src/components/Comments/CommentsSection.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+
+type CommentsSectionProps = {
+  restaurantId: string;
+};
+
+function CommentsSection({ restaurantId }: CommentsSectionProps) {
+  const [comments, setComments] = useState<{ text: string }[]>([]);
+  const [newComment, setNewComment] = useState<{ text: string }>({ text: "" });
+
+  useEffect(() => {
+    axios
+      .get(`/api/restaurants/${restaurantId}/comments`)
+      .then((response) => {
+        setComments(response.data);
+      })
+      .catch((error) => {
+        console.error("There was an error fetching the comments!", error);
+      });
+  }, [restaurantId]);
+
+  const handleAddComment = () => {
+    axios
+      .post(`/api/restaurants/${restaurantId}/comments`, newComment)
+      .then((response) => {
+        setComments(response.data.comments);
+        setNewComment({ text: "" });
+      })
+      .catch((error) => {
+        console.error("There was an error adding the comment!", error);
+      });
+  };
+
+  return (
+    <div>
+      <h3>Comments</h3>
+      <input
+        type="text"
+        value={newComment.text}
+        onChange={(e) => setNewComment({ text: e.target.value })}
```

Comment:
Please do not shorten words like event or response in order to make the code more readable. I know, a lot of people are shortening it on the internet.

Reply:
Please create and use a handleFunction here with a bit more logic. Also please create a name for this input field and use it's name in the handleFunctions logic.